lib/node_modules/@stdlib/blas/base/srotg/benchmark/benchmark.js

```diff
@@ -0,0 +1,80 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var Float32Array = require( '@stdlib/array/float32' );
+var pkg = require( './../package.json' ).name;
+var srotg = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var out;
+	var x;
+	var y;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*10.0 ) - 5.0;
+		y = ( randu()*10.0 ) - 5.0;
+		out = srotg( x, y );
+		if ( isnan( out[ i%4 ] ) ) {
+			b.fail( 'should not return NaN' );
+		}
+	}
+	b.toc();
+	if ( isnan( out[ i%4 ] ) ) {
+		b.fail( 'should not return NaN' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
+});
+
+bench( pkg+':assign', function benchmark( b ) {
+	var out;
+	var x;
+	var y;
+	var z;
+	var i;
+
+	out = new Float32Array( 4 );
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*10.0 ) - 5.0;
+		y = ( randu()*10.0 ) - 5.0;
+		z = srotg.assign( x, y, out, 1, 0 );
+		if ( typeof out !=='object' ) {
+			b.fail( 'should return an array' );
+		}
+	}
+	b.toc();
+	if ( z !== out ) {
```

Comment:
I'd used the L49 check here, as well, which is a bit more robust to potential future compiler optimizations.